src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgreSQLController.java

```diff
@@ -207,9 +158,18 @@ public void run()
         {
             while (true) {
                 final String newDbName = RandomStringUtils.randomAlphabetic(12).toLowerCase(Locale.ENGLISH);
-                create(pgDb, newDbName, "postgres");
+                SQLException failure = null;
+                try {
+                    create(pg.getPostgresDatabase(), newDbName, "postgres");
+                } catch (SQLException e) {
+                    failure = e;
+                }
```

Comment:
Could it just make more sense to throw an exception from here instead of having this orthogonal metadata added to the `DbInfo` object?  _Oh, never mind; I see that this is in a different thread adding to a queue._

The idea occurs to me that we could also have a superclass for the DB info stored in the queue, and then have subclasses instead of overloading the capabilities of the object.  But I suppose the usage here is small-scale enough that that would be overkill.